### PR TITLE
Updates for related to the transition to the new on-prem machines

### PR DESCRIPTION
**Pull request by frankhereford**
These are minor changes related to the way that these scripts are being setup on the new on-prem server. 

Testing isn't really going to be a thing here, so if you'll give it a good look and understand if I have to make some adjustments on Saturday. This is my best understanding of what we'll need though!

* 🎗️ Don't forget to rebuild and publish the images on merger!

### CODE REVIEW

File: scripts/restart.sh
Comment (chiaberry) [inline scripts/restart.sh:1]: should this be empty? or  `docker restart "cctv-images"` ?

Comment (frankhereford) [reply scripts/restart.sh:1]: I'm not sure if I'm following - the file's contents were not changed. I'm setting the executable bit for the file here.  Does that resolve your question?

image

Comment (chiaberry) [reply scripts/restart.sh:1]: so weird -- the github differences say "empty file" but when I view the file, i see the contents

Comment (chiaberry) [reply scripts/restart.sh:1]: Screenshot 2024-06-14 at 11 17 58 AM

**Review by chiaberry — APPROVED** [review]
i was gaslit by github. ✔️

File: scripts/restart.sh
Comment (frankhereford) [reply scripts/restart.sh:1]: I think this is right. I didn't make any changes to the contents of the file. I only set the executable bit on it -- so the shell will invoke the file if it's run. I did `chmod a+x restart.sh`.

**Comment by frankhereford** [comment]
@chiaberry wrote:
> i was gaslit by github. ✔️

Thanks for checking though!! I appreciate your review 🙏

File: scripts/restart.sh
Comment (mddilley) [reply scripts/restart.sh:1]: this makes sense! thanks Frank for pointing that out - GH's way of showing these kinds of permission changes always confuses me 🙃

**Comment by frankhereford** [comment]
Oh! @chiaberry - you absolutely did uncover something that I missed! I need to remove the `sudo` from that file -- and that would have shown up in the diff. Thank you!! https://github.com/cityofaustin/atd-cctv-images/pull/5/commits/422ce666271011d5c25e8e47d00cb4068df2fb70

**Review by mddilley — APPROVED** [review]
🚢 🙌 🚢

**Comment by frankhereford** [comment]
@mddilley wrote:

> this makes sense! thanks Frank for pointing that out - GH's way of showing these kinds of permission changes always confuses me 🙃

Yea, now that you say it, it is really bad. It should be something like `-rw-r--r--` => `-rwxr-xr-x`

**Pull request merged**